packages/tasks/src/pipelines.ts

```diff
@@ -635,6 +635,16 @@ export const PIPELINE_DATA = {
 		modality: "cv",
 		color: "yellow",
 	},
+	"text-to-3d": {
```

Comment:
Feel free to add `subtasks` for different outputs if you think it's worth it, but no need for now